.github/workflows/build.yml

```diff
@@ -1,9 +1,12 @@
-name: All Tests
+name: Build
 
 on: [push, pull_request]
```

Comment:
whoops, let's fix this in all four workflow files. it should run on all PRs and on push to main.